### PR TITLE
Hug 640 ability border in blocks

### PR DESCRIPTION
**Pull request by mkrishtopa**
This change is [Reviewable](https://reviewable.io/reviews/hugo-events/mosaico/40)

### CODE REVIEW

File: templates/versafix-1/template-versafix-1.html [hunk 1]
```diff
@@ -341,7 +356,7 @@
         frameTheme { label: Frame Style; }
         frameTheme:preview { -ko-background-color: @[backgroundColor] }
 
-        footerBlock { label: Footer Block; theme: contentTheme }
+        footerBlock { label: Unsubscribe Block; theme: frameTheme }
```
Comment (bubenkoff) [inline templates/versafix-1/template-versafix-1.html]: this has significant impact on background settings, are you sure?

**Review by bubenkoff — CHANGES_REQUESTED** [review]

File: templates/versafix-1/template-versafix-1.html (hunk 1 above)
Comment (mkrishtopa) [reply templates/versafix-1/template-versafix-1.html]: https://github.com/hugo-events/mosaico/compare/v0.15...v0.16#diff-775bd5c94e4cd6c3777b70b09cd333c9L112

Comment (bubenkoff) [reply templates/versafix-1/template-versafix-1.html]: no, we don't need to follow how it was, frameTheme is not flexible enough for blocks, all blocks have to have content theme

Comment (mkrishtopa) [reply templates/versafix-1/template-versafix-1.html]: yes, sir!

**Review by bubenkoff — APPROVED** [review]

**Pull request merged**